.github/workflows/tests-mailman.yaml

```diff
@@ -0,0 +1,36 @@
+name: Run tests (GNU Mailman 3)
+
+on:
+  # # Trigger the workflow on master but also allow it to run manually.
+  # workflow_dispatch:
+  # push:
+  #    branches:
+  #      - master
```

Comment:
do we want to make this run in every branch or only on master + manual as the documentation suggests?

Reply:
No, probably only master+manual. I've now converted this PR back to Draft until it's ready for review again.

Reply:
Ready for review now!